art/attacks/evasion/imperceptible_asr/imperceptible_asr_pytorch.py

```diff
@@ -85,22 +85,22 @@ class ImperceptibleASRPyTorch(EvasionAttack):
     def __init__(
         self,
         estimator: PyTorchDeepSpeech,
-        initial_eps: float = 0.001,
-        max_iter_1st_stage: int = 1000,
-        max_iter_2nd_stage: int = 4000,
-        learning_rate_1st_stage: float = 0.1,
-        learning_rate_2nd_stage: float = 0.001,
-        optimizer_1st_stage: Optional["torch.optim.Optimizer"] = None,
-        optimizer_2nd_stage: Optional["torch.optim.Optimizer"] = None,
-        global_max_length: int = 10000,
+        eps: float = 0.05,
+        max_iter_1: int = 10,
+        max_iter_2: int = 4000,
+        learning_rate_1: float = 0.001,
+        learning_rate_2: float = 5e-4,
+        optimizer_1: "torch.optim.Optimizer" = "torch.optim.Adam",
+        optimizer_2: "torch.optim.Optimizer" = "torch.optim.Adam",
```

Comment:
Did you intend `"torch.optim.Adam"` to be a string?

Reply:
It's fixed.